### DISCUSSION TITLE
How to start on Ubuntu20.04(WSL2)?

### DISCUSSION THREAD

**Discussion by galaxyzen** [johnfactotum/foliate#952] [General]
**Question:**
I have installed the foliate on Ubuntu20.04(WSL2), but there no shortcut in the windows start menu. How can I start using it?

**Version:**
 - foliate: 2.6.4-1~ubuntu20.04.1 all
 - OS/Distribution and version: Ubuntu20.04(WSL2)
 - Installation method: ppa

**Comment by galaxyzen** [comment]
I got it. Type `com.github.johnfactotum.Foliate` instead of `foliate` in the terminal. But I meet a new issue: There is no GUI environment on my Ubuntu20.04(WSL2). So how can I start it on my windows?
> Unable to init server: Could not connect: Connection refused
error: XDG_RUNTIME_DIR not set in the environment.
error: XDG_RUNTIME_DIR not set in the environment
Nested Wayland compositor could not create display socket
Unable to init server: Could not connect: Connection refused

**Reply by gvenusleo** [reply]
Maybe this is because of a lack of a driver for vGPU. To run Linux GUI apps, you should first install the driver matching your system. Just follow this [tutorial](https://learn.microsoft.com/en-us/windows/wsl/tutorials/gui-apps).
